common/v2/services/Store/Network/helpers.ts

```diff
@@ -11,9 +11,14 @@ import {
   NodeOptions,
   WalletName
 } from 'v2/types';
+import { makeExplorer } from 'utils/helpers';
 
 export const getAllNetworks = () => {
-  return Object.values(getCache().networks);
+  const networks = Object.values(getCache().networks);
```

Comment:
No direct access to cache (We're fighting to remove them).
Use the networks in the state of the `Provider`.

Reply:
I'm confused by your conflicting statements when reviewing this file (this one and the one above). Shouldn't I use `getNetworkByName` from this file to get network details?

The reason I had to run `makeExplorer` again is because the cache is stored in localStorage, and thus the functions `addressUrl` etc aren't available when I get the details of a network. 

Please let me know if there is another way I should be accessing network details where this is available.